src/Chapter2.hs

```diff
@@ -621,7 +652,11 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: Integral a => [a] -> [a]
```

Comment:
This function doesn't need a constrain on the `a` type variable. It works with all lists, as you don't use any specific information about elements 🙂